### DISCUSSION TITLE
/bottom-tabs Native blur effect?

### DISCUSSION THREAD

**Discussion by Ceriumin** [react-navigation/react-navigation#12031] [Q&A]
The native stack provides a way to render native headers and apply a blur effect when collapsing the header, however is there a way to achieve this effect with the bottom-tabs navigator.

This is an effect most notable on IOS applications and would like to replicate this effect. Thanks

**Comment by satya164** [comment]
https://reactnavigation.org/docs/bottom-tab-navigator#tabbarbackground

**Reply by Ceriumin** [reply]
> https://reactnavigation.org/docs/bottom-tab-navigator#tabbarbackground

Sorry I should've been more specific, the header has functionality which automatically blurs after collapsing, the tab bar remains static and has no option for such an effect
